.github/test-lib.yaml

```diff
@@ -0,0 +1,70 @@
+name: Run tests
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - 'library/**'
+  pull_request:
+    branches: [master]
+    paths:
+      - 'library/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - name: Checkout
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache library node modules
+        id: cache-npm-lib
+        uses: actions/cache@v3
+        with:
+          path: library/node_modules
+          key: ${{ runner.os }}-lib-${{ hashFiles('**/library/package-lock.json') }}
+
+      - name: Install library dependencies
+        run: npm ci --prefix library
+        if: ${{ steps.cache-npm-lib.outputs.cache-hit == 'true' }}
+
+      - name: Lint
+        run: npm run lint:check --prefix library
+
+      - name: Build
+        run: npm run build --prefix library
+
+      - name: Serve test pages
+        run: npm run serve --prefix library
+
+      - name: Cache extension node modules
+        id: cache-npm-ext
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+
+      - name: Install extension dependencies
+        run: npm ci
+        if: ${{ steps.cache-npm-ext.outputs.cache-hit == 'true' }}
+
+      - name: Compile Swarm Extension
+        run: npm run swarm-ext
+
+      - name: Build Blossom Extension
+        run: npm run build
```

Comment:
`--prefix` is not needed?

Reply:
No, because this command is executed in the root directory.